lib/api/apiUtils/bucket/parseWhere.js

```diff
@@ -40,6 +40,35 @@ const exprMapper = {
     'LIKE': '$regex',
 };
 
+function parseLikeExpression(value, operator) {
```

Comment:
Since this a method you put together(not attributed code) and in the spirit of keeping utilities modular, can you move this into another file and export it directly?